src/neuroglancer/datasource/precomputed/frontend.ts

```diff
@@ -242,7 +242,7 @@ function parseShardingEncoding(y: any): DataEncoding {
 }
 
 function parseShardingParameters(shardingData: any): ShardingParameters|undefined {
-  if (shardingData === undefined) return undefined;
+  if (shardingData === undefined || shardingData === null) return undefined;
```

Comment:
if (!shardingData) return;
is equivalent

Reply:
It's good enough as is tbh, if you don't feel like changing it I'll merge it in.

Reply:
Not exactly, that would also count `0, '', false, NaN` which would be more likely to trigger an exception right after.

Reply:
Would it be okay to merge this version? Thanks!